alphabase/psm_reader/dia_psm_reader.py

```diff
@@ -93,7 +88,7 @@ def __init__(  # noqa: PLR0913 many arguments in function definition
         )
 
 
-class DiannReader(SpectronautReader):
+class DiannReader(MaxQuantReader):
```

Comment:
Sorry if I get confused here :D 
Didn't we move to a new name for the MaxQuantReader as base tsv reader?